alot/crypto.py

```diff
@@ -196,17 +169,18 @@ def verify_detached(message, signature):
 
     :param message: the message as `str`
     :param signature: a `str` containing an OpenPGP signature
-    :returns: a list of :class:`gpgme.Signature`
+    :returns: a list of :class:`gpg.results.Signature`
     :raises: :class:`~alot.errors.GPGProblem` if the verification fails
     '''
-    message_data = StringIO(message)
-    signature_data = StringIO(signature)
-    ctx = gpgme.Context()
-
-    status = ctx.verify(signature_data, message_data, None)
-    if isinstance(status[0].status, gpgme.GpgmeError):
-        raise GPGProblem(status[0].status.message, code=status[0].status.code)
-    return status
+    ctx = gpg.core.Context()
+    try:
+        verify_results = ctx.verify(message, signature)[1]
```

Comment:
Where csn I find the docs for gpg to check what this does? (Especially the `[1]`)